src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ACTAccessRequirementWidget.java

```diff
@@ -149,6 +149,7 @@ public void onSuccess(String rootWikiId) {
     convertACTAccessRequirementButton.configure(ar, refreshCallback);
     view.setAccessRequirementName(ar.getName());
     view.setAccessRequirementID(ar.getId().toString());
+    view.setSubjectsDefinedByAnnotations(ar.getSubjectsDefinedByAnnotations());
```

Comment:
These changes are x4 (one for ACTAccessRequirement (legacy type), ManagedACTAccessRequirement, SelfSignAccessRequirement, and TermsOfUseAccessRequirement (legacy type)

Reply:
Very little shared code between these widgets :/

Reply:
The other widgets (ManagedACT AR, SelfSign AR, TermsOfUse AR) only `setSubjectsDefinedByAnnotations` if `isACT`. Just curious, why does the ACT AR widget handle setting the `subjectsDefinedByAnnotations` differently than the other AR widgets?

Reply:
Good question.  I think because ACT Access Requirements should no longer be used, so the UI is even lazier (users trying to gain access for this type of AR need to go out-of-band and will see some details in this renderer that may be confusing).  There's a migration path for ACT to move these to the Managed ACT AR type, which Kimyen added many years ago